src/styles/components/_syntax-highlighter.scss

```diff
@@ -135,23 +136,29 @@ pre > code.diff-highlight .token.inserted:not(.prefix) {
             font-size: var(--mynah-syntax-code-font-size) !important;
             font-family: var(--mynah-syntax-code-font-family) !important;
         }
+
         code {
             color: var(--mynah-color-syntax-code);
             filter: brightness(0.95);
             white-space: pre;
             background-color: inherit;
         }
 
-        > code,
-        & {
-            text-align: left;
-            white-space: pre;
-            word-spacing: normal;
-            word-break: normal;
-            word-wrap: normal;
-            tab-size: 4;
-            hyphens: none;
+        /**
+        * Apply the default styles to the code block.
+        * Should we decide to use word wrapping in the future, we can uncomment the following lines.
```

Comment:
We will not break words in the future for code blocks, because of the line numbers will be used on them if they'll come from the user's own context.

Reply:
However, i'll move this to UX board to get an approval for a toggle to switch between wrapped and unwrapped versions of them.

Reply:
Decision from the UX board:
Wrap if the available area is too narrow (up to ??px which will be given from the UX team) and otherwise keep horizontal scrolling.